lib/responsys/configuration.rb

```diff
@@ -3,17 +3,40 @@ class Configuration
     attr_accessor :settings
 
     def initialize
-      @settings = {
-        username: nil,
-        password: nil,
-        wsdl: "",
-        debug: false,
-        sessions: {
-          size: 80,
-          timeout: 30
+      @settings = {}
+    end
+
+    def savon_settings
+      settings_hash = if !@settings[:wsdl].blank?
```

Comment:
I feel like it's a very long line, 141 characters :/ 

Also your comment made me realize that the project is subject to be used outside of Rails but `.present?` and `.blank?` are in the Rails API and not in ruby itself which shouldn't work at all !?

After digging into the code it looks like Nori, the Savon dependency, monkey-patched `blank?` in class `Nori::CoreExt::Object`.

I'll add the patch in our gem since it lightens the code and we're probably gonna go toward a REST client and remove Savon.